server/models/transaction.js

```diff
@@ -8,6 +8,10 @@ module.exports = (sequelize, DataTypes) => {
       autoIncrement: false,
       defaultValue: DataTypes.UUIDV4,
     },
+    parentTransaction:{
+      type: DataTypes.UUID,
+      allowNull: false
```

Comment:
allowNull should prob be true since the first transaction (creating the challenge) will not have a parent

Reply:
I was presently using a constant, `const GENESIS_PARENT_UUID = "00000000-0000-0000-0000-000000000000";` for the challenge creation. Would this be better or worse than the suggestion of having no parent value?

Reply:
null makes more sense (no parent)